### PR TITLE
Feat/gno risk param

### PR DESCRIPTION
**Pull request by Rozengarden**

### CODE REVIEW

File: src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Update GNO Risk Parameters on Aave V3 Gnosis Pool"
+author: " Aave Chan Initiative"
```
Comment (sakulstra) [inline src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool.md:3]:
```suggestion
author: " Aave Chan Initiative"
```
```suggestion
author: "Aave Chan Initiative"
```

File: src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Update GNO Risk Parameters on Aave V3 Gnosis Pool"
```
Comment (sakulstra) [inline src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool.md:2]: perhaps better "Enable GNO borrowing"

File: src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool_20231213.s.sol [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ * verify-command: npx catapulta-verify -b broadcast/UpdateGNORiskParametersOnAaveV3GnosisPool_20231213.s.sol/100/run-latest.json
+ */
+contract DeployGnosis is GnosisScript {
+  function run() external broadcast {
+    // deploy payloads
+    AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213 payload0 = AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213(
+        GovV3Helpers.deployDeterministic(
+          type(AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213).creationCode
+        )
+      );
```
Comment (sakulstra) [inline src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool_20231213.s.sol:20]:
```suggestion
    address payload0 =
        GovV3Helpers.deployDeterministic(
          type(AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool_20231213).creationCode
        );
```

both is fine, but better align with patched generator if ppl check for reference

File: src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool.md (hunk 2 above)
Comment (eboadom) [reply src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool.md:2]: As there are some other params updated (e.g. borrow cap, RF), I would say better `Update GHO parameters on Aave V3 Gnosis Pool`, general enough

Comment (eboadom) [reply src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool.md:2]: Same to be applied on the payload file itself, not only AIP

**Review by eboadom — DISMISSED** [review]

**Review by sakulstra — APPROVED** [review]

**Pull request merged**